### DISCUSSION TITLE
大佬，现在对于英文单词缩写发音不正确要怎么处理，比如AI

### DISCUSSION THREAD

**Discussion by apzy** [fishaudio/Bert-VITS2#266] [General]
在三语言模型中，如AI，AR，HR这种缩写，现在是当作了一个单词发音，对于这种情况要怎么处理。

**Comment by ghost** [comment]
这种只能手动改成谐音字发音了吧 没有什么好的办法

**Comment by qieqieplus** [comment]
改改g2p，把两三个大写字母组成的又不在词典内的词按单字母发音

**Comment by RoversCode** [comment]
目前的音素处理，AI会变成音素 ai，所以要么在g2p那里做处理，要么推理的时候，手动分开 ，比如A，I。

**Comment by yin-zhang** [comment]
一个简单的办法是在g2p里面用LangSegment处理一下，就会把缩写自动分开。

**Comment by chj113** [comment]
请教下各位，具体代码怎么改动呢，可否给个示例？自己训练的语音模型，输入A-Z发音都不准，又要怎么解决？感谢回复~
